buildSrc/mac.gradle

```diff
@@ -55,7 +55,9 @@ def defaultSdkPath = "/Applications/Xcode.app/Contents/Developer/Platforms/MacOS
 
 // Set the minimum API version that we require (developers do not need to override this)
 // Note that this is not necessarily the same as the preferred SDK version
-defineProperty("MACOSX_MIN_VERSION", "10.10");
+def isAarch64 = TARGET_ARCH == "aarch64" || TARGET_ARCH == "arm64";
+def macOSMinVersion = (TARGET_ARCH == "aarch64" || TARGET_ARCH == "arm64") ? "11.0" : "10.12";
```

Comment:
I guess `isAarch64` was defined to use on line 59 in ternary operator.

Reply:
Fixed.